charts/shoot-core/components/charts/coredns/values.yaml

```diff
@@ -57,6 +57,8 @@ configmap:
     - name: reload
     - name: loadbalance
       parameters: round_robin
+    - name: import
+      parameters: custom/*.override
```

Comment:
should not it be? 
```suggestion
      parameters: custom/*.server
```

Reply:
Actually, these 2 lines should be removed.

Reply:
no its correct this way, *.override has a different purpose that *.server. with *.server you modify the plugins used by the main server in the config.

Reply:
Thanks!
I see that you've also added some info about this in the docs file, so now it makes sense.